test/flycheck-test.el

```diff
@@ -4489,6 +4489,24 @@ Why not:
      '(4 3 error "Expected indentation of 2 characters"
          :checker javascript-jscs))))
 
+(flycheck-ert-def-checker-test javascript-standard javascript error
+  :tags '(checkstyle-xml)
```

Comment:
Please remove this line.  This test isn't related to checkstyle currently, so the tag is wrong.